components/phpunit_bridge.rst

```diff
@@ -34,6 +34,12 @@ Alternatively, you can clone the `<https://github.com/symfony/phpunit-bridge>`_
 
 .. include:: /components/require_autoload.rst.inc
 
+
+.. best-practice::
+
+    PHPUnit Bridge is designed to remain compatible with Symfony version 2.7 up to master,
```

Comment:
Not sure we should tell about 2.7: it's being EOLed in a few weeks. Maybe something like that instead?
> The PHPUnit bridge is designed to work with all maintained versions of Symfony components, even across different major versions of them. You should always use its very latest stable major version to get the most accurate deprecation reports.

Reply:
Maybe also, the line above "$ composer require --dev symfony/phpunit-bridge" should be updated to
`$ composer require --dev "symfony/phpunit-bridge:*"`